### PR TITLE
Add Healthcheck

### PR DESCRIPTION
**Pull request by dpanayotov**

### CODE REVIEW

File: pkg/sbproxy/reconcile/reconcile_brokers.go [hunk 1]
```diff
@@ -50,13 +50,17 @@ type ReconcilationTask struct {
 
 // Settings type represents the sbproxy settings
 type Settings struct {
-	URL string
+	URL      string
```
Comment (dpanayotov) [inline pkg/sbproxy/reconcile/reconcile_brokers.go:53]: this shouldn't be `reconcile.Username`

File: pkg/middleware/basic_auth.go [hunk 2]
```diff
@@ -17,40 +17,61 @@
...
-					StatusCode:  http.StatusUnauthorized,
-				})
-				if err != nil {
-					logger.Error(err)
-				}
-				return
-			}
-			handler.ServeHTTP(w, r)
-		})
+func BasicFilter(username, password string) web.Filter {
```
Comment (dzahariev) [inline pkg/middleware/basic_auth.go]: [warning]: pkg/middleware/basic_auth.go: 32:1: exported function BasicFilter should have comment or be unexported (golint)

File: pkg/middleware/basic_auth.go [hunk 3]
```diff
@@ -17,40 +17,61 @@
...
+
+func (*noOpEncrypter) Decrypt(ctx context.Context, ciphertext []byte) ([]byte, error) {
+	return ciphertext, nil
+}
+
+type inMemoryCredentials struct {
+	username, password string
+}
+
+func InMemoryCredentials(username, password string) storage.Credentials {
```
Comment (dzahariev) [inline pkg/middleware/basic_auth.go]: [warning]: pkg/middleware/basic_auth.go: 63:1: exported function InMemoryCredentials should have comment or be unexported (golint)

**Comment by dzahariev** [comment]
@pankrator

**Review by pankrator — APPROVED** [review]

**Pull request merged**